install-unit

```diff
@@ -48,6 +49,7 @@ test -f _blacklist && {
 }
 
 # Exit if there's a guard file and the guard detects an install
+#what is guard?
```

Comment:
guard is a test.  you will see `guard_file` defined above in this file.  Installed.sh is the guard file.  It stops other units from installing when they are dependancies etc... feel free to rename it if you don't think it's clear